spec/arachnid2_spec.rb

```diff
@@ -113,7 +187,7 @@
       allow(File).to receive(:open).with(Arachnid2::MEMORY_USE_FILE, 'rb').and_return(use_file)
       allow(File).to receive(:open).with(Arachnid2::MEMORY_LIMIT_FILE, 'rb').and_return(limit_file)
 
-      expect(@spider.send(:memory_danger?)).to be_falsey
+      expect(dummy.send(:memory_danger?)).to be_falsey
```

Comment:
why do you need 'send' here, 'memory_danger?' is a public method of Exoskeleton module?